test-support/helpers/interactions.js

```diff
@@ -8,7 +8,7 @@ import {
 
 const { isEmpty } = Ember;
 
-export function checkByLabel(labelText) {
+export function clickByLabel(labelText) {
```

Comment:
`clickByLabel` clicks the label, `clickRadioByLabel` clicks the radio input. Any thoughts on this ?